bin/log-include.sh

```diff
@@ -12,10 +12,23 @@ if [ "$logVerbose" != "true" ]; then
   exec >/dev/null
 fi
 
-function add_notice {
+function add_notice_store {
```

Comment:
Why the new/additional function?  Does the following not work?
`echo "$1$(printf %$(eval 'echo $n')s |tr ' ' ' ')"  >> $TMP_DIR/notices.txt`

Reply:
The old **add_notice_wide** referenced **add_notice** and I was trying to preserve that.
Bryan helped me fix it this morning and should be addressed in a later push.